documentcloud/documents/tests/test_entity_extraction.py

```diff
@@ -4,252 +4,78 @@
+        {"text": {"content": "Professor", "begin_offset": 17434}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 17873}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 17863}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 26361}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 26351}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 26498}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 26488}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 27352}, "type_": 1},
+        {"text": {"content": "Knight", "begin_offset": 27899}, "type_": 1},
+        {"text": {"content": "Knight", "begin_offset": 30121}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 30111}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 30432}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 30422}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 31321}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 31311}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 31915}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 31905}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 32254}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 32244}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 32528}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 32518}, "type_": 2},
+        {"text": {"content": "Knight", "begin_offset": 43223}, "type_": 1},
+        {"text": {"content": "Professor", "begin_offset": 43213}, "type_": 2},
+    ],
 }
 
+
 @pytest.mark.django_db()
 class TestEntityExtraction:
-    def mock_bulk_create(self, entity_objs):
-        self.entity_objs_passed_to_bulk_create = entity_objs
-        print("entity_objs", entity_objs)
-
-    def test_get_or_create_entities(self):
+    def test_get_or_create_entities(self, mocker):
         """get or create entities"""
-        self.entity_objs_passed_to_bulk_create = None
-        #entity = Entity(**entity_dict)
-        #assert entity
-        #pdb.set_trace()
+        mock_bulk_create = mocker.patch(
+            "documentcloud.documents.entity_extraction.Entity.objects.bulk_create"
+        )
+
+        _get_or_create_entities([entity_dict])
 
-        # Mock
-        _get_or_create_entities([entity_dict], bulk_create=self.mock_bulk_create)
-        assert self.entity_objs_passed_to_bulk_create, "Entity objects were not passed to bulk_create."
-        assert len(self.entity_objs_passed_to_bulk_create) == 1, "The number of entity objects passed to bulk_create are incorrect."
-        entity_obj = self.entity_objs_passed_to_bulk_create[0]
-        #pdb.set_trace()
+        assert mock_bulk_create.call_args.args[
```

Comment:
This is in the documentation I sent you in more detail, but the mock object has a property called `call_args` which contains the arguments the mock was last called with - this contains both `args` and `kwargs`.  We want the first positional argument, hence `.call_args.args[0]`